scripts/zkai_native_attention_mlp_rmsnorm_label_sensitivity_gate.py

```
@@ -0,0 +1,588 @@
+
+def validate_payload(payload: dict[str, Any]) -> None:
+    if payload.get("schema") != SCHEMA:
+        raise RmsnormLabelSensitivityError("schema drift")
+    frontier = _dict(payload.get("frontier"), "frontier")
+    if frontier.get("frontier_win_claimed") is not False:
+        raise RmsnormLabelSensitivityError("frontier overclaim")
+    if frontier.get("nanozk_win_claimed") is not False:
+        raise RmsnormLabelSensitivityError("NANOZK overclaim")
+    summary = _dict(payload.get("summary"), "summary")
+    if _int(summary.get("best_label_probe_typed_bytes"), "best probe") != LABEL_PROBE_A_TYPED_BYTES:
+        raise RmsnormLabelSensitivityError("best label probe drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)